src/slider/stateful-slider-container.js

```diff
@@ -22,28 +22,23 @@ class StatefulSliderContainer extends React.Component<
   StateT,
 > {
   static defaultProps = {
-    initialState: {value: []},
+    initialState: {values: []},
     stateReducer: defaultStateReducer,
     onChange: () => {},
   };
 
   state = {...this.props.initialState};
 
-  onChange = (config: {event: *, value: Array<number>}) => {
-    const {event, ...params} = config;
-    this.internalSetState(STATE_CHANGE_TYPE.change, event, params);
+  onChange = (params: {values: Array<number>}) => {
+    this.internalSetState(STATE_CHANGE_TYPE.change, params);
     const {onChange} = this.props;
-    return onChange({event, ...params});
+    return onChange({...params});
```

Comment:
nit - do you mind not destructuring `onChange` from props here? it feels hard to follow when the method calling it is named the same.

also, something to add to the migration documentation is that we're no longer passing `event` to the `onChange` callback